syntax = "proto3";
package paxoskv;
message GetReq{
    bytes key = 1;
}
message GetResp{
    bytes value = 1;
}
message SetReq{
    bytes key = 1;
    bytes value = 2;
}
message SetResp{
}
message DelReq{
    bytes key = 1;
}
message DelResp{
}

service PaxosKVService {
    rpc Get (GetReq) returns (GetResp);
    rpc Set (SetReq) returns (SetResp);
    rpc Del (DelReq) returns (DelResp);
}

